__tests__/endpoints.test.js

```diff
@@ -1,35 +1,121 @@
+const res = require('express/lib/response');
 const request = require('supertest');
 const app = require('../app');
 const db = require('../db/connection');
 const testData = require('../db/data/test-data');
 const seed = require('../db/seeds/seed');
 
+// --== Hooks (Setup/Teardown) ==--
 beforeEach(() => seed(testData));
-
 afterAll(() => db.end());
 
-// Start of Tests
-describe('GET - /api/topics', () => {
-  test('returns array of topic objects', () => {
+// --== Tests ==--
+describe('API-wide tests', () => {
+  test('STATUS: 500, Returns error if code is broken)', () => {
+    return request(app)
+      .get('/api/brokenpath')
+      .expect(500)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe('Server Error');
+      });
```

Comment:
This should return a 404 just like in the following test. You cannot test for 500s